subjects/copy/README.md

```diff
@@ -29,19 +30,19 @@ pub fn vec_function(b: Vec<i32>) -> (Vec<i32>, Vec<f64>) {
 
 ### Usage
 
-Here is a possible program to test your function :
+Here is a possible program to test your function:
 
 ```rust
 use copy::*;
 
 fn main() {
-    let a: i32 = 0;
-    let b = String::from("1 2 4 5 6");
-    let c = vec![1, 2, 4, 5];
+    let a = "1 2 4 5 6".to_owned();
+    let b = vec![1, 2, 4, 5];
+    let c = 0;
 
-    println!("{:?}", nbr_function(a));
-    println!("{:?}", str_function(b));
-    println!("{:?}", vec_function(c));
+    println!("{:?}", nbr_function(c));
+    println!("{:?}", vec_function(b));
+    println!("{:?}", str_function(a));
 }
 ```
 
```

Comment:
The output here in the README is different from the one in the solution:
In the solution, we get this with the new changes:
```
(0, 1.0, -inf)
([1, 2, 4, 5], [0.0, 0.6931471805599453, 1.3862943611198906, 1.6094379124341003])
("1 2 4 5 6", "2.718281828459045 7.38905609893065 54.598150033144236 148.4131591025766 403.4287934927351")
```

Basically switch line 54 and 5.